src/components/templates/top/index.tsx

```diff
@@ -0,0 +1,35 @@
+import { useQuery } from 'react-query';
+import { Category, fetchCategories } from './fetcher';
+import { SimpleCard } from '../../molecules/SimpleCard';
+import { Headline } from '../../atoms/Headline';
+import styles from './styles.module.scss';
+
+export function Top() {
+  const { data, isLoading, error } = useQuery<Category[]>(
+    'categories',
+    fetchCategories,
+  );
+
+  if (isLoading) {
+    return <>Loading...</>;
+  }
+
+  if (error) {
+    return <>An error has occurred: {error}</>;
+  }
```

Comment:
🔔 [should]
エラー時に Error オブジェクトが throw されてくるため、`<>{error}</>` で怒られてしまっています 👀 

次のように対処すると良いと思います！

```suggestion
  const { data, isLoading, error } = useQuery<Category[], Error>(
    'categories',
    fetchCategories,
  );

  if (isLoading) {
    return <>Loading...</>;
  }

  if (error) {
    return <>An error has occurred: {error.message}</>;
  }
```